js/form.js

```diff
@@ -1,17 +1,92 @@
-const toInactiveForm = (form, inactiveClass) => {
-  form.classList.add(inactiveClass);
-  const elements = form.children;
-  for (let i = 0; i < elements.length; i++) {
-    elements[i].disabled = true;
+const adForm = document.querySelector('.ad-form');
+const adTitle = adForm.querySelector('#title');
+const adPrice = adForm.querySelector('#price');
+const adType = adForm.querySelector('#type');
+const adRoomNumber = adForm.querySelector('#room_number');
+const adCapacity = adForm.querySelector('#capacity');
+const adTimein = adForm.querySelector('#timein');
+const adTimeout = adForm.querySelector('#timeout');
+
+const toggleFormState = (form, inactiveClass) => {
+  form.classList.toggle(inactiveClass);
+  const elements = form.elements;
+  for (const element of elements) {
+    if (element.disabled) {
+      element.disabled = false;
+    } else {
+      element.disabled = true;
+    }
   }
 };
 
-const toActiveForm = (form, inactiveClass) => {
-  form.classList.remove(inactiveClass);
-  const elements = form.children;
-  for (let i = 0; i < elements.length; i++) {
-    elements[i].disabled = false;
+const setMinPrice = (type) => {
+  switch (type) {
+    case 'bungalow':
+      return '0';
```

Comment:
```suggestion
    case 'bungalow': {
      return '0';
      }
```
добавь всем кейсам курли